src/main/java/Davvy.java

```diff
@@ -0,0 +1,110 @@
+import java.util.Scanner;
+
+import static java.lang.Integer.parseInt;
+
+public class Davvy {
+    private static final String BOT_NAME = "Davvy";
+    private static boolean toEnd;
+    public static void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    private static void printStatement(String statementType) {
+        printLine();
+        switch (statementType) {
+        case "greetings":
+            System.out.println(" Hello! I'm " + BOT_NAME + "\n" + " What can I do for you?");
+            break;
+        case "goodbye":
+            System.out.println(" Bye. Hope to see you again soon!");
+            break;
+        }
+        printLine();
+    }
+
+    public void startChat() {
+        printStatement("greetings");
+        toEnd = false;
+        Scanner in = new Scanner(System.in);
+        while (!toEnd) {
+            String[] parsedInput = processInput(in.nextLine());
+            processCommand(parsedInput);
+        }
+    }
+
+    public void endChat() {
+        printStatement("goodbye");
+    }
+
+    public String[] processInput (String inputText) {
+        if (inputText.isEmpty()) {
+            printLine();
+            System.out.println("Please type something >:(");
+            printLine();
+            return null;
+        }
+        return inputText.split("\\b", 2);
```

Comment:
Can include comment to help readers with what `\b` regex is

Reply:
I will do just that, thanks!